src/core/l10n.js

```diff
@@ -17,7 +17,7 @@ const base = {
   editors_note: "Editor's note",
   editors: "Editors:",
   example: "Example",
-  fig: "Fig. ",
+  fig: "Figure ",
```

Comment:
(You can probably teach your screenreader to recognise it, but I think that's pushing the work to the wrong place :) ).

Thanks, BTW